### ISSUE TITLE
Go to new chat page when invite link is visited

### ISSUE THREAD

**Issue by agazso** [logos-innovation-lab/waku-objects-playground#23]

**Comment by agazso** [comment]
Closed by #89

**Comment by vojtechsimetka** [comment]
I don't think this is resolved by #89 but I am happy to not implement this.

**Comment by agazso** [comment]
You are right that technically the #89 does not solve this issue. However as part of the discussions about the invites there was consensus in that changing screens when someone else is doing an action may not be the best user experience and we are looking for alternative solutions.
